fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2634,6 +2634,7 @@ public Range getUnbuiltRange() {
                             throw wrapped;
                         } else {
                             updateIndexState(index.getName(), indexKey, IndexState.READABLE);
+                            clearReadableIndexBuildData(tr, index);
```

Comment:
Does this need to be added to `uncheckedMarkIndexReadable`, too, or was this not done on purpose (so that force marking an index as `READABLE` doesn't clear out this information)?

Reply:
It seems that the cases of `uncheckedMarkIndexReadable` calls that will leave behind temporary index building information are extreme and rare enough to have this data staying behind. If anything, it'll make a useful debug information (for example, seeing which part of the record store was actually indexed before enforcing a readable state).